### PR TITLE
Hunter is SUS SUS AMOGUS

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Stealth will transform the hunter into it's marked target (everything but a human).  FUN CONFIG only

https://user-images.githubusercontent.com/25491240/132957360-9c7a1310-dd8b-4c1f-bfe0-c928391d06ac.mp4

## Why It's Good For The Game

SUS

## Changelog
:cl:
add: Hunter can disguise itself as it's marked target, this replaces stealth (fun config)
/:cl:

### CODE REVIEW

**Comment by bobmancer343** [comment]
HUNTER VENTING IN IMPOSTER AMONG US

**Comment by joniroq** [comment]
Can't wait to get bwoinked for shooting other marines and say I was "spy checking" them

**Comment by mcmeiler** [comment]
> 
> 
> Can't wait to get bwoinked for shooting other marines and say I was "spy checking" them

Do it with a flamethrower as well for maximum funny

**Comment by neohenry** [comment]
time to play prop HUNTer

**Comment by SplinterGP** [comment]
is this some random fun config feature with a balance change

**Comment by BraveMole** [comment]
> is this some random fun config feature with a balance change

Nah i just forgot to remove the balance part, it was there initially (when this wasn't a shitpost)

**Comment by Tristan63** [comment]
kek

**Comment by Ragecrabbo** [comment]
I need this

File: config/config.txt [hunk 1]
```diff
@@ -221,3 +221,5 @@ DEFAULT_VIEW_SQUARE 15x15
 
 ## Once a typepath causes overrun from hard deletes this many times, stop hard deleting it on garbage collection failures. (set to 0 to disable)
 #HARD_DELETES_OVERRUN_LIMIT 0
+
+FUN_ALLOWED 1
```
Comment (TiviPlus) [inline config/config.txt:225]:
```suggestion
#HARD_DELETES_OVERRUN_LIMIT 0
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**